.storybook/webpack.config.js

```diff
@@ -37,6 +37,12 @@ module.exports = ({ config }) => {
   });
   config.resolve.extensions.push('.tsx');
   config.resolve.extensions.push('.ts');
+  /* eslint-disable-next-line no-param-reassign */
+  config.resolve.alias = {
```

Comment:
need this to ensure we never have multiple versions of react being used in SB

Reply:
Is this a risk given the version of Node we are on should be deduping or does it still happen regardless?

Reply:
To be honest, I don't understand 100% why there would be multiple copies of react. Running `npm ls react` or `npm ls react-dom` does show only one version and one copy, my assumption is that it's some issue with Storybook